GVFS/GVFS.Common/ProductUpgraderInfo.Shared.cs

```diff
@@ -36,14 +37,9 @@ public static bool IsLocalUpgradeAvailable(ITracer tracer)
             return false;
         }
 
-        public static string GetHighestAvailableVersionFilePath()
-        {
-            return Path.Combine(GetUpgradesDirectoryPath(), HighestAvailableVersionFileName);
-        }
-
-        public static string GetUpgradesDirectoryPath()
+        public static string GetHighestAvailableVersionFilePath(string upgradesDirectory)
```

Comment:
Now that these static methods take extra arguments, should we make them into their own class? With tests?